### DISCUSSION TITLE
random disconnects

### DISCUSSION THREAD

**Discussion by Pshemas** [Ylianst/MeshCentral#6293] [Q&A]
One of the clients, which admittedly has quite poor internet connection, randomly disconnects. When I check error log on the agent I see:

![obraz](https://github.com/user-attachments/assets/b963d4aa-73d2-4da6-8dbc-4c32e8dd5486)

I wonder - is there anything I can do to limit such events?

**Comment by Raito00** [comment]
Hello!

I think "Agent Ping and Browser Pong" can help.

Nice info here:
https://www.youtube.com/watch?v=xoR3FoqUnlA
